docs/source/mods/sharpe-ratio.rst

```diff
@@ -0,0 +1,224 @@
+
+    We seek a portfolio of weights :math:`x` that maximizes the Sharpe ratio :math:`(\mu^\top x - r_f) / \sqrt{x^\top \Sigma x}`:
+
+    .. math::
+        \begin{alignat*}{2}
+            \max_x\ && \frac{\mu^\top x - r_f}{\sqrt{x^\top \Sigma x}} & \\
+            \textrm{s.t.}\ && \sum_{i=1}^n x_i ={} &1 \qquad \textbf{(1)} \\
+            && x \geq {} &0.
+        \end{alignat*}
+
+    Model :math:`\textbf{(1)}` is non-convex. In general, non-convex problems are very difficult to solve. Fortunately, there exists a convex reformulation of :math:`\textbf{(1)}`. As a step towards that convex reformulation, we first reformulate :math:`\textbf{(1)}` as follows:
+
+    .. math::
+        \begin{alignat*}{2}
+            \max_y\ && \frac{1}{\sqrt{y^\top \Sigma y}} \qquad & \\
+            \textrm{s.t.}\enspace && (\mu - r_f)^\top y = {} &1 \qquad \textbf{(2)} \\
+            && y \geq {} &0.
+        \end{alignat*}
+
+    Models :math:`\textbf{(1)}` and :math:`\textbf{(2)}` are equivalent in the sense that given a solution to either problem, we can construct a solution to the other of equal or better objective value. In particular, any solution :math:`\bar{x}` of :math:`\textbf{(1)}` can be mapped to a solution :math:`\bar{y}` of :math:`\textbf{(2)}` of equivalent objective value using the transformation :math:`\bar{y}_i := \bar{x}_i / \mu^\top \bar{x}` for :math:`i = 1, \ldots, n`. Conversely, any solution :math:`\bar{y}` of :math:`\textbf{(2)}` can be mapped to a solution :math:`\bar{x}` of :math:`\textbf{(1)}` of equivalent objective value using the transformation :math:`\bar{x}_i := \bar{y}_i / \sum_{j = 1}^n \bar{y}_j` for :math:`i = 1, \ldots, n`.
+
+    Like :math:`\textbf{(1)}`, model :math:`\textbf{(2)}` is non-convex. However, because :math:`\Sigma` is positive semidefinite, maximizing :math:`1 / \sqrt{y^\top \Sigma y}` is equivalent to minimizing :math:`y^\top \Sigma y`. Thus, the optimal solution of :math:`\textbf{(2)}` is equivalent to the optimal solution of the following model:
+
+    .. math::
+        \begin{alignat*}{2}
+            \min_y\ && y^\top \Sigma y \qquad \quad & \\
+            \textrm{s.t.}\enspace && (\mu - r_f)^\top y = {} &1 \qquad \textbf{(3)} \\
+            && y \geq {} &0.
+        \end{alignat*}
+
+    Model :math:`\textbf{(3)}` is convex. In this OptiMod, we solve the convex quadratic program (QP) :math:`\textbf{(3)}`, then map the optimal solution :math:`y^*` back to the original problem :math:`\textbf{(1)}` via the transformation :math:`x^*_i := y^*_i / \sum_{j=1}^n y^*_j` for :math:`i = 1, \ldots, n`.
```

Comment:
Same as above, but here I think that the final transformation is important enough to warrant display rather than inline style. Then it could also be a "real" fraction without the small font problem.

Reply:
Good idea. I moved the final transformation to display style.